### PR TITLE
fix typo on custom-types.mdx

### PR DESCRIPTION
**Pull request by Proxiweb**
typo fix

### CODE REVIEW

File: src/pages/language-tour/custom-types.mdx [hunk 1]
```diff
@@ -228,7 +228,7 @@ They are most useful for simplifying type signatures.
 type Person = (String, Integer)
 
 fn create_person(name: String, age: Integer) -> Person {
-    #(name, age)
+    (name, age)
```
Comment (KtorZ) [inline src/pages/language-tour/custom-types.mdx:231]: Hey! The typo is actually on the line above, it should be:

```
type Person = #(String, Integer)
```

Comment (Proxiweb) [reply src/pages/language-tour/custom-types.mdx:231]: It's a bit weird no ? The primitive tuple is declared without the # ...

Comment (rvcas) [reply src/pages/language-tour/custom-types.mdx:231]: It is not atm

Comment (KtorZ) [reply src/pages/language-tour/custom-types.mdx:231]: Oopsie. We didn't update the doc regarding tuples.

**Review by KtorZ — APPROVED** [review]

**Pull request merged**